File: src/main/java/jp/artan/japanesefoodmod/common/world/type/WorldTypeFruitsForest.java
package jp.artan.japanesefoodmod.common.world.type;

import net.minecraft.world.World;
import net.minecraft.world.WorldType;
import net.minecraft.world.biome.BiomeProvider;

public class WorldTypeFruitsForest extends WorldType {
    public WorldTypeFruitsForest() {
        super("Fruits Forest");
    }

    @Override
    public BiomeProvider getBiomeProvider(World world)
    {
        return new BiomeProviderFruitsForest();
    }

}

File: src/main/java/jp/artan/japanesefoodmod/common/world/type/WorldTypeFruitsForest.kt
package jp.artan.japanesefoodmod.common.world.type

import net.minecraft.world.World
import net.minecraft.world.WorldType
import net.minecraft.world.biome.BiomeProvider

class WorldTypeFruitsForest : WorldType("Fruits Forest") {
    override fun getBiomeProvider(world: World): BiomeProvider {
        return BiomeProviderFruitsForest()
    }
}
